src/deviceApiCalls/schemas/sendJSPixel.params.json

```diff
@@ -4,25 +4,50 @@
   "type": "object",
   "description": "Send pixels data to be fired from the native layer",
   "additionalProperties": false,
-  "properties": {
-    "pixelName": {
-      "oneOf": [
-        {
+  "oneOf": [
```

Comment:
Changed signature, to strongly type extra params for each pixel, now `pixelName` field acts as discriminator.